idaes/surrogate/pysmo/sampling.py

```diff
@@ -371,29 +473,23 @@ def __init__(self, data_input, number_of_samples=None, sampling_type=None):
         print('Sampling type: ', self.sampling_type, '\n')
 
         if self.sampling_type == 'selection':
-            if isinstance(data_input, pd.DataFrame):
-                data = data_input.values
-                data_headers = data_input.columns.values.tolist()
-            elif isinstance(data_input, np.ndarray):
-                data = data_input
-                data_headers = []
+            if isinstance(data_input, pd.DataFrame) or isinstance(data_input, np.ndarray):
```

Comment:
Trivial improvement: the second argument of `isinstance()` can also be a tuple, so this can be written `isinstance(data_input, (pd.DataFrame, np.ndarray))`.

Reply:
Done.